node/network/approval-distribution/src/metrics.rs

```diff
@@ -25,9 +25,8 @@ struct MetricsInner {
 	assignments_imported_total: prometheus::Counter<prometheus::U64>,
 	approvals_imported_total: prometheus::Counter<prometheus::U64>,
 	unified_with_peer_total: prometheus::Counter<prometheus::U64>,
-	aggression_l1_messages_total: prometheus::Counter<prometheus::U64>,
-	aggression_l2_messages_total: prometheus::Counter<prometheus::U64>,
-
+	// aggression_l1_messages_total: prometheus::Counter<prometheus::U64>,
+	// aggression_l2_messages_total: prometheus::Counter<prometheus::U64>,
```

Comment:
it would be good to extract aggression levels removal into a separate PR

Reply:
I will actually keep the code as I improved it in a different PR.